Syllabus.md

```diff
@@ -0,0 +1,80 @@
+# RoboJackets Software Training - Fall 2021
+
+The RoboJackets software training program is designed to prepare you to work on software for the RoboJackets competitive teams. This program offers practical experience writing code using C++ and ROS. It also covers some of the fundamental concepts of robotics used by all of our teams.
+
+## Trainers
+
+- Vivek Mhatre
+- Matthew Hannay
+- Nicolas Leone
+- Isaac Dale
+- Charlie Jenkins
+- Hussain Gynai
+- Kyle Stachowicz
+- Kevin Fu
+- Adithya Vasudev
+
+## Meeting Schedules
+
+TODO
+
+## Resources
+
+- [Software Training Mailing List](https://lists.gatech.edu/sympa/robojackets-training-sw)
+
+  An email list for official training-related announcements.
+
+- [Training Calendar](https://robojackets.org/calendar/action~month/cat_ids~12/request_format~html/)
+
+  See details for all RoboJackets training meetings.
+
+- [Piazza](https://piazza.com/class/j4pry2dzzg76pl)
+
+  Get your questions answered any time by the instructors or other students.
+
+- [RoboJackets Training YouTube Channel](https://www.youtube.com/channel/UCh3TLV-vQzzcWGQ4u2jsMOw)
+
+  Where all training videos will be posted.
+  
+- [Software Training Repository](https://github.com/RoboJackets/software-training)
+
+  The GitHub repository that hosts most of the resources for the software training program, including project instructions and starter code.
+
+- [STSL Repository](https://github.com/RoboJackets/stsl)
+
+  The Software Training Support Library repository. This holds support code for the training projects. Open issues against this repository if you find bugs with the robots or simulator.
+
+## Prerequisites
+
+We will assume that students are familiar with the concepts covered in [AP Computer Science A](https://apstudents.collegeboard.org/courses/ap-computer-science-a). Topics we assume knowledge of will be briefly covered in Week 0 content. Topics covered in AP Computer Science A that have special syntax, properties, or behavior in C++ will be covered in the main course content.
+
```

Comment:
Add a section about the pre-reps for the robotics track

"Students should be familiar with AB level calculus. Namely the concepts of a derivatives and integrals."